frontend/src/layout/panel-layout/ProjectTree/ProjectTree.tsx

```diff
@@ -111,6 +111,8 @@ export function ProjectTree({
     const { projectTreeMode } = useValues(projectTreeLogic({ key: PROJECT_TREE_KEY }))
     const { setProjectTreeMode } = useActions(projectTreeLogic({ key: PROJECT_TREE_KEY }))
 
+    const showFilterDropdown = root === 'project://'
```

Comment:
This could even be a prop of `<ProjectTree />` to split out the logic a bit, but this works too